components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
```

Comment:
add color var here

Reply:
All colors are actually included in variables in the following format:
`$color-colorNameHere`

- To use the "Campus Gold", you would use the SCSS variable `$color-campusGold`.
- To use the "Fountain Run Teal", you would use the SCSS variable `$color-fountainRunTeal`

Use the full color name for each brand color, listed on the [colors page](http://www.purdue.edu/brand/visual/colors.html).

A list of color variables can be found within [_base.scss](https://github.com/PurdueMarketingAndMedia/purdueTemplates-2015/blob/master/components/css/global/_base.scss).